styles/main.scss

```diff
@@ -59,39 +59,12 @@
     content: "";
     z-index: 6000;
   }
-  
+
   /* FAB label in dark mode */
   .theme-dark .fab-label {
     background-color: #333;
     color: white;
   }
-  
-  /* Tabs standalone issue */
-  .tabs-standalone {
-    height: auto;
-  }
-  
-  .popup {
-    overflow-y: auto;
-  }
-  
-  .swiper {
-    margin-top: 20px;
-    width: 90%;
-  }
-  
-  .theme-dark .swiper-slide {
-    background-color: #1c1c1d;
-  }
-  
-  .swiper-slide {
-    border-radius: var(--f7-card-border-radius);
-    min-height: 400px;
-  }
-  
-  .light .swiper-slide {
-    background-color: white;
-  }
```

Comment:
@DivadNojnarg this has impact on the swiper component, I'll fix that now

Reply:
@hypebright. For standalone tabs, you can check with this example to check whether things break:

```r
shinyApp(
   ui = f7Page(
     title = "My app",
     f7SingleLayout(
       navbar = f7Navbar(
         title = "Standalone tabs"
       ),
       f7Tabs(
         id = "tabs",
         style = "strong", animated = FALSE, swipeable = TRUE,
         f7Tab(
           tabName = "Tab1",
           icon = f7Icon("envelope"),
           active = TRUE,
           f7Card(
               title = "Card header",
               f7Stepper(
                 "obs1",
                 "Number of observations",
                 min = 0,
                 max = 1000,
                 value = 500,
                 step = 100
               ),
               plotOutput("distPlot")
             )
         ),
         f7Tab(
           tabName = "Tab2",
           icon = f7Icon("today"),
           f7Card(
               title = "Card header",
               f7Select(
                 inputId = "obs2",
                 label = "Distribution type:",
                 choices = c(
                   "Normal" = "norm",
                   "Uniform" = "unif",
                   "Log-normal" = "lnorm",
                   "Exponential" = "exp"
                 )
               ),
               plotOutput("distPlot2")
             )
         ),
         f7Tab(
           tabName = "Tab3",
           icon = f7Icon("cloud_upload"),
           f7Card(
               title = "Card header",
               f7SmartSelect(
                 inputId = "variable",
                 label = "Variables to show:",
                 c("Cylinders" = "cyl",
                   "Transmission" = "am",
                   "Gears" = "gear"),
                 multiple = TRUE,
                 selected = "cyl"
               ),
               tableOutput("data")
             )
         )
       )
     )
   ),
   server = function(input, output) {
     output$distPlot <- renderPlot({
       dist <- rnorm(input$obs1)
       hist(dist)
     })

     output$distPlot2 <- renderPlot({
       dist <- switch(
         input$obs2,
         norm = rnorm,
         unif = runif,
         lnorm = rlnorm,
         exp = rexp,
         rnorm
       )

       hist(dist(500))
     })

     output$data <- renderTable({
       mtcars[, c("mpg", input$variable), drop = FALSE]
     }, rownames = TRUE)
   }
 )
```

Reply:
@DivadNojnarg that looks all good 👍 

For the Swiper it's not so bad either: the swipe controls are much higher, but this actually makes much more sense because they need to be on the element. And the element just happens to be not-so-high. So I'm only updating the example to look a bit better, but the classes gone are an improvement

Reply:
Great. These were to fix issues 3 years from now. It totally makes sense if this is not needed anymore because these are now fixed in the core Framework7.